docs/en/basic_configuration.md

```diff
@@ -48,6 +48,26 @@ class MyFormPage extends Page
 }
 ```
 
+## Control when child/parent pages are included in cache actions
+
+There are two configurations available, and they can both be set to one of three available values:
+
+* `regenerate_children`
+  * `none`: Do not include any children
+  * `direct`: Include only one level below (direct children, but not grandchildren, etc)
+  * `recursive`: Include all children recursively (grandchildren, great-gc, etc)
+* `regenerate_parents`
+  * `none`: Do not include any parents
+  * `direct`: Include only one level above (the direct parent)
+  * `recursive`: Include all parents recursively (up to the top of the SiteTree)
```

Comment:
```suggestion
## Control when child/parent pages are regenerated in cache actions

There are two configurations available, and they can both be set to one of three available values:

* `regenerate_children`
  * `none`: Do not regenerate any children
  * `direct`: Regenerate only one level below (direct children, but not grandchildren, etc)
  * `recursive`: Regenerate all children recursively (grandchildren, great-gc, etc)
* `regenerate_parents`
  * `none`: Do not regenerate any parents
  * `direct`: Regenerate only one level above (the direct parent)
  * `recursive`: Regenerate all parents recursively (up to the top of the SiteTree)
```
I've noticed that the word 'include' is being used alongside the word 'regenerate' as a synonym.  I found this confusing and it's far easier to understand what's going on if we simply replace the word 'include' with 'regenerate' for everything here